### ISSUE TITLE
Help CMD broken in dms

### ISSUE THREAD

**Issue by mebrooks01** [BuildTheEarth/main-bot#289]
help cmd broken in dms
seems to be reading a non-existent cache
```log
 TypeError: Cannot read properties of undefined (reading 'cache')
        at Function.hasRole (/home/bots/main-bot/dist/struct/discord/GuildMember.js:16:28)
        at /home/bots/main-bot/dist/commands/help.js:41:89
        at Map.filter (/home/bots/main-bot/node_modules/@discordjs/collection/dist/index.js:1:2645)
        at Command.run (/home/bots/main-bot/dist/commands/help.js:41:49)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async Client.default_1 (/home/bots/main-bot/dist/events/messageCreate.js:76:13)
```

**Comment by mebrooks01** [comment]
fixed waiting to push commit

**Comment by mebrooks01** [comment]
fixed in pr #290
